### DISCUSSION TITLE
NPM Unable to connect from local network, Public Access works fine

### DISCUSSION THREAD

**Discussion by ingar195** [NginxProxyManager/nginx-proxy-manager#2547] [Q&A] [answered]
Hi
I have some domains that public-facing and some I just want on my local network
All my public facing one works perfectly

The problems start with my local ones. Have added rules for my subdomains, but still have no access
if I just switch it over to the Public Access List it works.

Does anyone have any clues?

**Comment by Budman1758** [comment, marked as answer]
Local DNS may be to blame here. Actually, the lack of it. Had the same issues until I added entries in my Pi-Hole local DNS records

**Reply by ingar195** [reply]
do you have an example of how you added to pi-hole?

**Reply by ingar195** [reply]
After some digging.
I have my ISP router in bridge mode and my UDM as my router but since that's not the most outer part of my network, it seems that the isp router is the one doing the hairpin.

will just setup pi-hole so I can manage my DNS locally

**Reply by Budman1758** [reply]
![image](https://user-images.githubusercontent.com/6570134/213238772-c0903a0f-6041-410c-97c3-ceccd6ebad6a.png)

Entries go here in Pi-Hole

**Reply by ingar195** [reply]
Hi 
I used this guide so all my subdomains will be redirected in stead of creating one for each and have to update 2 places
https://hetzbiz.cloud/2022/03/04/wildcard-dns-in-pihole/

**Reply by Budman1758** [reply]
Nice!  Thanks.
